src/ext/oblivc/ot.c

```diff
@@ -677,6 +677,31 @@ senderExtensionBoxNew (ProtocolDesc* pd, int destParty, int keyBytes)
 
   return s;
 }
+SenderExtensionBox*
+senderExtensionBoxSplit (ProtocolDesc * pd, SenderExtensionBox* t)
```

Comment:
Should `SenderExtensionBox` be `const`? I guess we are changing state of t->keyblock[i] by reading from it, but C won't complain about const-ness since those are referenced through a pointer.

Reply:
Maybe? Should this be a matter of convention (i.e. communicating in the fn signature that the passed struct won't be altered) or of admissibility (anything that can be const should be). If the latter, I suspect there are a lot of things that aren't currently const but should be. If the former, I would argue that reading from the keyblock probably doesn't count (in terms of what we're trying to communicate) as a change, but resetting it probably does count (which seems to be what we're going to do). Thoughts?